### ISSUE TITLE
avy--with-avy-keys deprecated

### ISSUE THREAD

**Issue by arbelt** [syl20bnr/spacemacs#5307]
After a recent update of avy, `spacemacs/ace-buffer-links` no longer works since `avy--with-avy-keys` is no longer defined.

Found this on the `avy` page: https://github.com/abo-abo/ace-link/issues/21

For now, I'm working around this with:

```elisp
  (with-eval-after-load 'avy
    (when (and (not (fboundp 'avy--with-avy-keys))
              (fboundp 'avy-with))
     (defalias 'avy--with-avy-keys 'avy-with)))
```

but should probably be changed in spacemacs.  Thanks for the great work!

**Comment by bjesus** [comment]
I just installed Spacemacs for the first name - this really makes it's hard to begin. I'm stuck on the first page, where it says that `o` would allow me to click on the top buttons, but `o` just gives the above error and so I can't even go and read the documentation.

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
